sites/automationworld/server/components/blocks/featured-companies.marko

```diff
@@ -0,0 +1,25 @@
+import queryFragment from "../../graphql/fragments/featured-companies";
+
+$ const { contentId } = input;
+
+<marko-web-query|{ nodes }|
+  name="related-published-content"
+  params={ contentId, limit: 20, queryFragment, includeContentTypes: ["Company"] }
+>
+  <marko-web-node-list class="mt-3">
+    <@header>Companies in this article</@header>
+    <@nodes nodes=nodes>
+      <@slot|{ node, index }|>
+        <marko-web-node type=`${node.type}-content` flush=true>
+          <@body>
+            <@title tag="h5">
+              $ const website = /^http/.test(node.website) ? node.website : `https://${node.website}`;
```

Comment:
Can this test be added to the GraphQL resolver for this field? Then it will be fixed globally.